software/esp-firmware/throttleHandling.cpp

```diff
@@ -194,7 +194,7 @@ void setLEDblink(unsigned int number)
   if(number > 0)
   {
     blinkLED = number;
-    ledStopOnTime = 100;
+    ledStopOnTime = 10;
```

Comment:
Also, this is the bugfix you are referring to in the commit message, right?

Heiko